2019_spd/sunpy-1p0.ipynb

```diff
@@ -0,0 +1,441 @@
3REGEr1F9KMKS4xUo92I7i53Dg9f+vK245A6Icm+PjuOfaAV5EKntKleFHytk8g4UDtpTZRVj0tbaIGQjwUdYxrU5Dc3y3mJfKhWXdB+33i2vH/+eCDtrCla4dqZF2jfRpgKUXH7wY9Fq9XkPIw+DLUm/4kWg3ZyT83kq52scVhcV1ZTZKKnZjGPGTs2lJUTdKarXep1RyPpHaxe86KZYioRxrVe1A6k/CwmTSsL0ftCxLXdFeJA1od6T5PFb9OH8c1zmeJxgyecAarCcK/O7U31jN1H2A3ECZmQcrn/24V76Agd97+oP7GLEGfSzAUmQNW1eKj3q9NYCeKqxRTxDiEz8SECztuFtKBa7UBH2ER9QRlW9N0yIqI4TDkxVghW8pfI0fKqljjjst/zHGLhSTjM+ivIhu/SuK9HPvrlarX1ytVv/uarX6N1er1bcl/arCuv2zn1E9XrvCzAZgw9JgkAOkpKvWodTMJVrTdj4TdAuGY3McZNtScyDG34hFxzt+NE9mAKADvMPd3XW9EftITcl1XXpqLq7sneo68OSaHXQd9NyvUFtvvsi0P2AcExoKJR4DB4PTTWW4w3XqTSmtd+hst9axIJhscD+8/hxvbi+Cw+syUgAm13ChJojX4Mpnle1QLkuZ5Bx/JAXq0Ett8fR3dxsp6BisJ6WOPgDT9lgM1Kkn6V63pHE7shN8UpmNglb14qD4Iop2PpK6cW03huE77A8S7G7slwQE+5LmzTbYOwoQ3DsKS3NR/LnTifTgo2zDbiv7U+3DpvgqHeuq2bYeFnOpTNpPOxJMX8fnugUH8wFV3lMTIKSrIiCnJGdKipaX+5r3d3f1ge2/sM88U1hu3pXaijHCFzzH8t1R0q48J+dK8HPh3C0FyBZ2XB/oanKUXjn3h2omMx/a/1PlclzEkH4WiQVe1IX/o9Vqdc08VFqtVh8rcrn+yBdi/qAAeOeh8G24kd7Rar+c1AQkwGtZbe8dmAKIULBmmOk/UXQw95+4MKbOzlNbWOzjfk5fXJnrw+foafiY6brFxEDb1fWWFA9d1/6jjV1lSpvx8NMmxP8x6x+qKTpyIPa2mqvpV/F2hh7CwqbNHQCxEGvKmMLkBHWf+7RYAklKmrUenLEAsOyIP+sprKTNfgFYU7peVweK+88YEBbV7xRv357SWuc0h/dDsKOukj79pEJspIPh6EHGQ37SPrPROlykfyRNjuP98e8V4Lun9Q2ZfJS7Lubhm338YViPpPKbjOI13M0Fq89O89oR4OyV62RdUCYQKFvPlGFJMBlMuJ6quVA1zwrb4N/DonNrExoSWtNBE6ue4/n5sSjb9htuHQ+p2lRamlLeS08J52AvZeA+z6f7uznOuZLyZJJ6oqug4z72tpJuhfbnOqGgTxXjGO1JPXsKwJ0qQko+0PXP3x+1vMiS/J1Wq/Wrn8E5vtSFoGGAzwGEAZkB3i04kpYDEvVkn+1cPANAAkCex5TfsOToHJ4lB4n3dQMvAHl6fHxFpEJh9sl5uKa2cqLgVpfK+VzNyzE98w7XVk8GOAYgdGnHkJrWG37emXIQ4HqZuTqdtaeUw9fUMbGLblG6tVA/eCiLa+EOgwVt4+IsLE3CW7DgmAAtlCnf2N4tCAZhfDTua7osOxLfh3XlghwfhJeSRsfHDQqcoPZabettBUAySA4VScD3DpVJA15UPGMOZXjnxfttDCM0ZK714sr93Qj7IISjdiJPxjFhmM+0TiAw3As/5Pe/E+1z8lg6fxKhNFidWJFS0GY+iJ8orUgWBKddoSFvKUHD+0NXKS5xrQGWFJYZoNNWAgGhEYQS1SEZznD37D9XdfMckxXn5Ph4PcFs2/6eHxkKuLaaVfYB/F35i8/ySXX9nmFIajI+PtnlP0Jfnqq58gfWL1mGpHRN/FMl+P+g5UUg+a9K+g9ardbfarVaX/sMzvWlKT+vpEsZtC2T1hoYPKerCzkQnQAMteXkVAgqUCwYAIbC/v4b1lxtIXk8ooOQW4ydTqcxELpF6WErHrIBKDt4uHDAFb2Aea3Evc7X6P8x7tVCGYoDp5SS/EvlOnyo65h97yksAILCa0EV1ueBAlCHak5EoJtoXygiQoBmyrAf9/86S8BANVbOqGm7jpqhKOzHZ0CNQW2sHOABStlvHh7DgLUePDudRjvWYTdSTjZctU38W0/SW4fSnXcKzTl+CW3fxlC6/SmiHi+jB9J3f1dSribCosvtdliC68w+ZZt5SfS+mAe1ev44fJHf/0783h9KJw8izIMJxck4/WDct4fKgf5EGSdKOVb2Max7wAVxCZMKZ0J6yrR23H/cCZ7eDlD0VHFYdjAL3NelEmz4T2q6J7iuHUk3Op01QzBTM7wKwMeS6yljkH1yiO/xVEF58sw9sXrQXp4gAMBEpEObMCnwtHY+NpJfgkkD/RmwPKy2/0HKJ4LkarX67mq1+mVJvyHp/261Wv9Hq9X6G7w+g3O/1uVMeQNcuYll5f4tRDxsQ+C9lDcS8ER1OlU6z5lx1opSrERWDWHgZiD0m0vdOMYzNZN1A7b7+/uNsJLrQh54IP2aPU6QbXmIpMxt6v/dsH1ox+tmqq4ylV07s+jaB0PSASkfUB7KOqYL0cGt6hx+rYgTeNWFtujZd3/3CQWWI20IUFKvp8pQCsQMvuByrTT1gfBSEa7QbsfrsozKy0W8+q3cr6bHNvb3GwPqjpJOBbC3lAAJgLt1dPedksgcfuzTLMk/SmGFkKMDiRVBTqUnvytNzsMveWMYCzRLksZqJFUYn4cVOS9ASHaihgp4HC+SPUC1uiU4ikOv/V5ThRgMCw6wcHHLsOx/oAwPq8Ve3idZKcYVsrQ9fQSrS0qLklU/0Eq4hefhHLgQAMaN/f2173JfCYao6qUEKtirHTUTE1yn0QL0AT1nJkiawDV52ApUMOdwrsH7KGJJZ4SInaQNLyT9in6w8sIQkFar9WOS/qKkvyvp1/XZULyvfQEgXZzDslXue6SxfM
```

Comment:
This plot is off at the moment - you need to add np.deg2dat(this_coord.lon)

Reply:
Thanks!